src/Controls/src/Core/Compatibility/Android/FontNamedSizeService.cs

```diff
@@ -112,21 +112,22 @@ public double GetNamedSize(NamedSize size, Type targetElementType, bool useOldSi
 			return defaultValue;
 		}
 
+		global::Android.Content.Context _applicationContext;
 		bool TryGetTextAppearance(int appearance, out double val)
 		{
 			val = 0;
 			try
 			{
 				using (var value = new TypedValue())
 				{
-					var context = Forms.MauiContext.Context;
-					if (context.Theme.ResolveAttribute(appearance, value, true))
+					_applicationContext ??= global::Android.App.Application.Context.ApplicationContext;
+					if (_applicationContext.Theme.ResolveAttribute(appearance, value, true))
```

Comment:
Does the theme change per activity? Maybe not for typical apps, but _can) it change per activity or even a fragment/view?

Reply:
The old code was just using the `Application.Context` as well so this is symmetrical with how it's been working. If we want this to work per Activity then we should do that in a new PR